spec/features/stations/user_can_create_a_station_spec.rb

```diff
@@ -3,6 +3,7 @@
 describe "When a user visits '/stations/new'" do
   it "they can create a new station" do
     visit 'stations/new'
+    save_and_open_page
```

Comment:
@riverswb No, I forgot to take it out. I just fixed the failing test so they should all be good now